acstech/doppler-events
# Producer and Client Working Together

The producer has been added to the API server that the user will connect to.

Fixes #36 
The node files were removed from the repository.
Fixes #4 
The backend can connect to kafka via a producer and the data is written to the database.

To test:
1) start up docker compose (`docker-compose up -d`).
2) run the API server.
3) run the test user once or twice. 
4) check influx for the data (`docker-compose exec influx_db influx`, `USE doppler`, `SELECT * FROM dopplerDataHistory`). There should entries for each time you ran the test user (the clientID should be test2).
